### DISCUSSION TITLE
Support for python 3.12

### DISCUSSION THREAD

**Discussion by Its-a-me-Ashwin** [isl-org/Open3D#7006] [General]
Is there a plan to release this library for python3.12?

**Comment by christian-rauch** [comment]
It should be included in the upcoming 0.19 release: https://github.com/isl-org/Open3D/pull/6717.

**Reply by futurewasfree** [reply]
Do you have an approximate release date in mind? Do you aim at 2024?

**Comment by Danilrivero** [comment]
Following https://github.com/isl-org/Open3D/pull/6717 will the release date for PyPI be anytime soon? Thanks in advance
